### ISSUE TITLE
Crash on Debian 12

### ISSUE THREAD

**Issue by pierrexn** [wang-bin/mdk-sdk#171]
When trying to play a video file, my app crashes on a fresh Debian 12. The bin tools from mdk package works...
It's the same with 0.23 & 0.24. 

Here is the logs, and stack. Do you have an idea?
[logs.txt](https://github.com/wang-bin/mdk-sdk/files/13849024/logs.txt)
![Capture-1](https://github.com/wang-bin/mdk-sdk/assets/1200371/b768a0e5-971f-4245-af92-a865baa8f810)

**Comment by wang-bin** [comment]
glfwplay works? I need that log. Is it x11 or wayland?

**Comment by pierrexn** [comment]
glfwplay works, it's wayland

**Comment by wang-bin** [comment]
what about glfwplay log?

**Comment by pierrexn** [comment]
[logs.txt](https://github.com/wang-bin/mdk-sdk/files/13854018/logs.txt)

**Comment by wang-bin** [comment]
you are using software decoder in glfwplay. to test vaapi: glfwplay -c:v VAAPI a.mp4

**Comment by wang-bin** [comment]
you are running software opengl. are you in a vm or no gpu driver installed?  i also need the output of `vainfo -display x11`

**Comment by pierrexn** [comment]
yes it's a VM, but it seems the same crash for user on OpenSUSE Tumbleweed KDE

**Comment by wang-bin** [comment]
what about vainfo -display x11?

**Comment by pierrexn** [comment]
![Capture-1](https://github.com/wang-bin/mdk-sdk/assets/1200371/cc153426-bf4a-45b4-86bf-e59d33dd3635)

**Comment by wang-bin** [comment]
vainfo does not crash, it's weird. run glfwplay -logile va.log -c:v VAAPI test.mp4

**Comment by pierrexn** [comment]
Whe using VAAPI, no more video, here is the logs
[va.log](https://github.com/wang-bin/mdk-sdk/files/13863205/va.log)

**Comment by wang-bin** [comment]
yes but no crash, it's expected. i don't known why crash in qt app
